test/unit/helpers/application_helper_test.rb

```diff
@@ -75,4 +75,52 @@ class ApplicationHelperTest < ActionView::TestCase
       assert_instance_of String, flash_message(:notice, @message)
     end
   end
+
+  context "avatar" do
+    setup do
+      @user = build(:user, email: "email@example.com")
+    end
+
+    should "raise when invalid theme is requested" do
+      assert_raises(StandardError) { avatar(160, "id", @user, theme: :unknown) }
+    end
+
+    context "with publicly available email" do
+      setup do
+        @user.public_email = true
+      end
+
+      should "return gravatar" do
+        url = avatar(160, "id", @user)
+        expected_uri = "https://secure.gravatar.com/avatar/5658ffccee7f0ebfda2b226238b1eb6e.png?d=retro&amp;r=PG&amp;s=160"
+
+        assert_equal "<img id=\"id\" width=\"160\" height=\"160\" src=\"#{expected_uri}\" />",
+url
```

Comment:
Is this supposed to be on the same line?

Reply:
uhh, rubocop automated fix, I'll fix it